src/storage/mvcc/txn.rs

```diff
@@ -72,8 +87,10 @@ impl<S: Snapshot> MvccTxn<S> {
         self.writes
     }
 
-    pub fn get_statistics(&self) -> &Statistics {
-        self.reader.get_statistics()
+    pub fn take_statistics(&mut self) -> Statistics {
```

Comment:
maybe reserve `get_statistics()`, seems that somewhere still need reference.

Reply:
The place where using it as a reference should use `take_statistics` instead 🤣It is just wrong to add twice and this interface will prevent such mistakes. Anyway this is a change from the refinement PR and after refinement we have no something like `get_statistics` at all 🤣

Reply:
Maybe in the future we can add something like `fn merge(&mut self, stat: Statistics)` to `Statistics` as an alternative to the method `add`, which consumes another `Statistics` but not to take its reference, making its semantic stricter.

Reply:
Agree